src/main/java/com/condominium/online/condo/controller/ApartmentManagerController.java

```diff
@@ -3,30 +3,38 @@
 import com.condominium.online.condo.entity.ApartmentManager;
 import com.condominium.online.condo.exceptions.InvalidUserException;
 import com.condominium.online.condo.service.ApartmentManagerService;
+import io.swagger.annotations.*;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.dao.DataIntegrityViolationException;
 import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 @RestController
+@RequestMapping("/condo/apartmentManager")
+@Api("v1 - Apartment Manager")
 public class ApartmentManagerController {
 
     private ApartmentManagerService apartmentManagerService;
 
     @Autowired
-    public ApartmentManagerController(ApartmentManagerService apartmentManagerService){
+    public ApartmentManagerController(ApartmentManagerService apartmentManagerService) {
         this.apartmentManagerService = apartmentManagerService;
     }
 
-    @PostMapping("/apartmentManager")
-    @ResponseStatus(code = HttpStatus.CREATED)
```

Comment:
Did you run the tests to make sure it's not broken ?
Removind this line might break some tests